issuetracker/src/main/java/org/openjdk/skara/issuetracker/Issue.java

```diff
@@ -188,6 +188,50 @@ default URI nonTransformedWebUrl() {
 
     List<Link> links();
 
+    /**
+     * Get the links which have the relationships listed in the arguments.
+     * @param relationships the link relationship
+     * @return the related links
+     */
+    default List<Link> linksWithRelationships(List<String> relationships) {
```

Comment:
I'm not sure there will ever be a need to filter links with multiple different relationships. Do you know of any such case that you are planning to convert?

The rest of this method could be converted to streams like this:

```
return links().stream()
    .filter(l -> link.relationship().isPresent() && relationships.contains(link.relationship().get())
    .toList();
```